src/UI/Components/Diagram/components/FormModal.tsx

```diff
@@ -158,7 +170,7 @@ const FormModal = ({
     const tempPossibleForms = getOptions(services, [
       {
         key: "default_option",
-        value: "Choose a Service",
+        value: undefined,
```

Comment:
Is this to be able to clear the select? If that's the case, default_option doesn't seem like the right key to me. Also, I wonder if we really do need that option at all

Reply:
After some thought I agree, removed